core/src/main/java/org/fao/geonet/resources/CMISConfiguration.java

```diff
@@ -508,69 +507,91 @@ public void init() {
-        }
-        // IncludeRelationships should be NONE
-        if (client.getDefaultContext().getIncludeRelationships().equals(IncludeRelationships.NONE)) {
-            Log.debug(Geonet.RESOURCES, "Changing default CMIS operational context to not include relationships.");
-            client.getDefaultContext().setIncludeRelationships(IncludeRelationships.NONE);
+                // Setup default options
+                // Ensure caching is on.
+                if (!client.getDefaultContext().isCacheEnabled()) {
+                    Log.debug(Geonet.RESOURCES, "Changing default CMIS operational context cache to enabled.");
+                    client.getDefaultContext().setCacheEnabled(true);
+                }
+                // Don't get allowable actions by default
+                if (client.getDefaultContext().isIncludeAllowableActions()) {
+                    Log.debug(Geonet.RESOURCES, "Changing default CMIS operational context to not include allowable actions.");
+                    client.getDefaultContext().setIncludeAllowableActions(false);
+                }
+                // Don't get ACLS by default
+                if (client.getDefaultContext().isIncludeAcls()) {
+                    Log.debug(Geonet.RESOURCES, "Changing default CMIS operational context to not include acls.");
+                    client.getDefaultContext().setIncludeAcls(false);
+                }
+                // Don't include path segments by default
+                if (client.getDefaultContext().isIncludePathSegments()) {
+                    Log.debug(Geonet.RESOURCES, "Changing default CMIS operational context to not include path segments.");
+                    client.getDefaultContext().setIncludePathSegments(false);
+                }
+                // Don't include policies by default
+                if (client.getDefaultContext().isIncludePolicies()) {
+                    Log.debug(Geonet.RESOURCES, "Changing default CMIS operational context to not include policies.");
+                    client.getDefaultContext().setIncludePolicies(false);
+                }
+                // IncludeRelationships should be NONE
+                if (client.getDefaultContext().getIncludeRelationships().equals(IncludeRelationships.NONE)) {
+                    Log.debug(Geonet.RESOURCES, "Changing default CMIS operational context to not include relationships.");
+                    client.getDefaultContext().setIncludeRelationships(IncludeRelationships.NONE);
+                }
+            } catch (CmisRuntimeException | CmisConnectionException e) {
+                client = null;
+                Log.error(Geonet.RESOURCES,
+                    "CMIS error creating session using base url \"" + repositoryUrl + "\" with repositoryName=\"" + repositoryName +
+                        "\". " + (e.getErrorContent() == null?"":"  Got following results from cmis api call:" + e.getErrorContent()), e);
```

Comment:
Please add a space after `call:` and format the `?` operator with the required spaces.